multi-layer-perceptron/KNet_test.jl

```diff
@@ -0,0 +1,143 @@
+using MLDatasets: MNIST
+using Knet, IterTools, MLDatasets
+using Dictionaries
+using TimerOutputs
+using TimerOutputs
+using JSON
+using Printf
+using Knet:minibatch
+using Knet:minimize
+using Knet
+using Knet: Param
+using Knet: Knet, dir, accuracy, progress, sgd, gc, Data, nll, relu
+using Flatten
+using Flux.Data;
+using Flux, Statistics
+
+# This loads the MNIST handwritten digit recognition dataset. This code is based off the Knet Tutorial Notebook. 
+xtrn,ytrn = MNIST.traindata(Float32)
+xtst,ytst = MNIST.testdata(Float32)
+println.(summary.((xtrn,ytrn,xtst,ytst)));
+
+xtrn = reshape(xtrn, 784, 60000 ) 
+xtst = reshape(xtst, 784, 10000 )
+println(summary.((xtrn, xtst))) # can see the data that is flattened 
```

Comment:
I'd remove these prints in the final commit as they are not needed in the final script. 

You could also set something up like a debug flag or logger so that if you debugging you can call these prints up, but if you are going that route I would add more to the messages to make it clear in the message what is happening.